ai-service/src/ai_service/__init__.py

```
@@ -1,0 +1,47 @@
+    "errors",
+    "utils",
+    # Main app
+    "app",
+    "main",
+    # Submodules
+    "embeddings",
+    "db_setup",
+    "handlers",
+    # Version
+    "__version__",
+]
```

Ruff (0.12.2)

`__all__` is not sorted

Apply an isort-style sorting to `__all__`

(RUF022)